ControlSettings/ControlSettingsTV.cpp

```diff
@@ -0,0 +1,3920 @@
+/*
+* If not stated otherwise in this file or this component's LICENSE file the
```

Comment:
I hope this code is identical from tvsetting thunder plugins

Reply:
Yes. It same as tvsettings-plugins.
I have make this as more generic and moved all the computation logic to HAL layer.